pkg/expr/commands.go

```diff
@@ -17,6 +17,7 @@ import (
 
 // Command is an interface for all expression commands.
 type Command interface {
+	fmt.Stringer
```

Comment:
Can we do a `Name() (string)` or something like this instead? Not sure we want to make String() the command name, and this would keep the specifics of what Unique names expectes and outputs decoupled from that.